account_payment_return_import_iso20022/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "summary": """
         This addon allows to import payment returns from ISO 20022 files
         like PAIN or CAMT.""",
-    "version": "12.0.1.1.0",
+    "version": "13.0.1.1.0",
     "development_status": "Production/Stable",
```

Comment:
You can change it to mature:
```suggestion
    "development_status": "Mature",
```